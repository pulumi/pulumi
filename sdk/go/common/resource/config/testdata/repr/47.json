{"my:outer":{"existing":"existingValue","inner":"value"}}