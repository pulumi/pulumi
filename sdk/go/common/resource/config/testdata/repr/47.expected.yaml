decrypt:
  my:outer: '{"existing":"existingValue","inner":"value"}'
paths:
  my:outer:
    value:
      existing: existingValue
      inner: value
    string: '{"existing":"existingValue","inner":"value"}'
    redacted: '{"existing":"existingValue","inner":"value"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAA8/4I5AAIIZXhpc3RpbmcGc3RyaW5nDA8ADWV4aXN0aW5nVmFsdWUFaW5uZXIGc3RyaW5nDAcABXZhbHVl
    secure: false
    isObject: true
  my:outer.existing:
    value: existingValue
    string: existingValue
    redacted: existingValue
    object: GhAABnN0cmluZwwPAA1leGlzdGluZ1ZhbHVl
    secure: false
    isObject: false
  my:outer.inner:
    value: value
    string: value
    redacted: value
    object: EhAABnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: false
